main/src/ca/uwaterloo/flix/language/ast/WeededAst.scala

```diff
@@ -224,7 +224,9 @@ object WeededAst {
 
     case class NewObject(tpe: Type, methods: List[JvmMethod], loc: SourceLocation) extends Expr
 
-    case class StructGet(exp: Expr, label: Name.Label, loc: SourceLocation) extends Expr
+    case class StructGet(struct: Expr, label: Name.Label, loc: SourceLocation) extends Expr
```

Comment:
We tend just to call the expressions for exp1, exp2, and so forth. Trying to keep the names consistent across 28 compiler phases is too much work :)

Reply:
You can fix it later.